doc/guides/contributing/pull-requests.md

```diff
@@ -630,4 +605,5 @@ More than one subsystem may be valid for any particular issue or pull request.
 [guide for writing tests in Node.js]: ../writing-tests.md
 [hiding-a-comment]: https://help.github.com/articles/managing-disruptive-comments/#hiding-a-comment
 [https://ci.nodejs.org/]: https://ci.nodejs.org/
+[pull request template]: https://raw.githubusercontent.com/nodejs/node/master/.github/PULL_REQUEST_TEMPLATE.md
```

Comment:
nit: maybe use relative path for folks who are reading the guide offline?
```suggestion
[pull request template]: ../../../.github/PULL_REQUEST_TEMPLATE.md
```

Reply:
Oops, I landed this but I'm going to force push it out because it links to the rendered version of the template which has a lot of stuff invisible because it is in comments.